### ISSUE TITLE
If bit64 isn't loaded, print.data.table() should warn if there are any integer64 columns present.

### ISSUE THREAD

**Issue by mattdowle** [Rdatatable/data.table#975]
Suggested by Stephen McInerney in person after Bay Area R User Group meetup, Nov 2014.

It's a simple if() and would avoid apparent "garbage/corruputed" data when in fact all that's needed is to require(bit64) for correct printing.

**Comment by arunsrinivasan** [comment]
Trying to link in the file in #810, this is what I get:

```
# Warning message:
# In fread("810_fread.txt") :
#   Some columns have been read as type 'integer64' but package bit64 isn't loaded. Those columns will display as strange looking floating point data. There is no need to reload the data. Just require(bit64) to obtain the integer64 print method and print the data again.
```
What doesn't this warning address?

**Comment by mattdowle** [comment]
It seems this warning was already here for a long time,  at least 2012.
I guess then, and combined with [this report](http://stackoverflow.com/questions/33268155/possible-bug-in-data-table-binary-search#comment54338457_33268155), the suggestion was to add a warning in the print method too.  Maybe integer64 columns were loaded from disk, created some other way, or the fread() warning simply missed.  The overhead of checking the class attribute of all columns when bit64 is not loaded, shouldn't be a speed concern since it's only printing.

**Comment by arunsrinivasan** [comment]
Fixed in commit 9362c4d161. Closes #975.
